stable/traefik/README.md

```diff
@@ -104,3 +104,13 @@ installing the chart. For example:
 ```bash
 $ helm install --name my-release --namespace kube-system --values values.yaml stable/traefik
 ```
+
+### Clustering / High Availability
+
+Currently it is possible to specify the number of `replicas` but the implementation is very naive.
+**Full Traefik clustering not yet supported.**
+i.e. Traefik master election is not yet supported. It is heavily advised to not set this value if you also have Let's Encrypt configured.
+
+While setting `replicas` will work for many cases, since no master is elected it has the consequence 
+that each node will end up requesting Let's Encrypt certificates if this is also configured.
+This will quickly cut into the very modest rate limit that Let's Encrypt enforces. 
```

Comment:
Can we merge this paragraph with the one above, since they're both brief and both are supporting the same general bit of advice? Might need to re-write a little after combining them.